src/main/controllers/style-guide/EtFortnightlyListController.ts

```diff
@@ -37,6 +37,7 @@ export default class EtFortnightlyListController {
                 venueTelephone: fileData['venue']['venueContact']['venueTelephone'],
             };
             const returnedCourt = await locationService.getLocationById(metaData['locationId']);
+            const courtRegion = req.lng === 'cy' ? returnedCourt.welshRegion : returnedCourt.region;
```

Comment:
Could we make the variable name here consistent between this controller and the daily (Happy with either one)